traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -98,11 +86,49 @@ RETURNING
 	(SELECT tm_user.username
 	 FROM tm_user
 	 WHERE tm_user.id=job_user) AS createdBy,
-	keyword,
-	parameters,
+	'PURGE' AS keyword,
+	CONCAT('TTL:', ttl_hr, 'h') AS parameters,
 	start_time
 `
 
+// Almost the same as insertQuery, but returns appropriate values for API 4.0+
+const insertQueryV40 = `
```

Comment:
We should probably call this `V4` instead of `V40`, because we'll likely just update this query with new fields if they're ever added.